essay.tex

```diff
@@ -0,0 +1,53 @@
+\title{Violence in games}
+
+\author{James Pitt}
+
+\date{\today}
+
+\documentclass[12pt]{article}
+
+\begin{document}
+\maketitle
+The difference in the DOOM controversy
+
+\begin{abstract}
+The video game industry has become the most prominent market in the world, generating more revenue last year than both the music and video industry. Developers for video games aim for audiences of all ages, some of which are of the more mature and violent nature.
+There have been strong cases on both sides of the argument as to whether video games incite violent tendencies in people. In 2010 CA Anderson published an article stating that there would seem to be a link between people playing video games and an increase in violent behaviour.
+
+In this article I will look at why the first DOOM in 1993 caused an uproar of controversy whereas the updated DOOM in 2016 caused little more than a small outburst. Also any correlation between density of video game sales and rate of crime and whether online multiplayer functionality helps reduce personal confrontation. Some scholars are arguing that video games such as Mortal Kombat and DOOM are a danger to public health as they increase aggression in younger people and encourage youth to replicate what they see on the screen.
+\end{abstract}
+
+
+\section{Influence on society}
+Video games have quickly become more relevant in the modern day world than books as a survey suggests that in Australia, 2011, 92% of homes had access to a video gaming console
+~\cite{Martin:2011:PGE:2071536.2071566}. Children are easily swayed by what they see and read as they have very little life experiences to compare them to and put them into contrast, therefore anything portrayed in the media that is accessible to children should make it very clear where to draw the line.
+Some extreme points of view state that games like DOOM train teenagers and children to be able to fight at a military grade level, with examples of the Colorado shootings in the 1990s and the anti religious sentiment surrounding the killings dubbing the game as a “Mass Murder Simulator”~\cite{videogamescankill}.
+The General Aggression Model believes that prolonged exposure to violent video games leads to an increase in general aggressive behaviour and higher expected hostility from people~\cite{Ashbarry:2016:BVE:2967934.2968111}~\cite{Fumhe:2015:VGP:2815782.2815790}. The critics of this statement say that it's down to the way the individual perceives it rather than the blanket statement that violent video games lead to a rise in aggression~\cite{5369086}. One of the reasons behind why they propose violent video games increase aggression is that players are faced with real life dilemmas with fictional or unreasonable solutions, an example of this is Grand Theft Auto V where the main character requires money so to solve the problem they rob a bank~\cite {Zendle:2015:HGF:2793107.2793113} ~\cite {Gotterbarn}.
+
+\section{DOOM 1993-2000}
```

Comment:
You need to consider the structure of your essay much more deeply. This section does not flow from the previous paragraph. I suggest booking a one-on-one meeting with the ASK team to assist you with this. Developing a bullet point outline of the key points you want to make in your argument is a good first step.